suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -15,11 +15,7 @@ type WalletItemProps = {
 export const WalletItem = ({ onPress, deviceState, isSelectable = true }: WalletItemProps) => {
     const device = useSelector((state: any) => selectDeviceByState(state, deviceState));
     const selectedDevice = useSelector(selectSelectedDevice);
-    const fiatBalance = useSelector((state: any) =>
-        device?.state?.staticSessionId
-            ? String(selectDeviceTotalFiatBalanceNative(state, device?.state?.staticSessionId))
-            : undefined,
-    );
+    const fiatBalance = useSelector(selectSelectedDeviceTotalFiatBalance);
```

Comment:
This is wrong, you can't show the fiat balance of the selected device for all devices... You have to select it by the staticSessionId

Reply:
good catch, reverted